cmd/go-httpbin/.gitignore

```diff
@@ -0,0 +1 @@
+/go-httpbin
```

Comment:
Not sure why this is necessary, but I'd prefer all ignores to be consolidated in the top-level `.gitignore` file.

If you use `make`/`make run`/`make watch`/`make test` during development, this probably wouldn't be needed!

Reply:
I moved it to toplevel gitignore. I added it so I do not risk to add it with `git add .` if if I manually build with `go build` inside the `cmd/go-httpbin` directory. But if you prefer I remove it. Sure it is not needed for the prefix.